index.js

```diff
@@ -61,28 +71,28 @@ class ScrollTrigger {
 
   inView() {
     const { className } = this.options;
+
     if (className && this.el.classList) {
       addClass(this.el, className);
     }
+
     const image = this.options.image;
+
     if (image) {
       if (this.el.tagName === 'IMG') {
-        if (this.el.getAttribute('src')) {
-          return;
-        }
-
-        addAttrs(this.el, { src: image });
+        this.el.setAttribute('src', image);
       } else {
-        if (this.el.style.backgroundImage) {
-          return;
-        }
-
         styles(this.el, {
           backgroundImage: `url(${image})`,
           backgroundRepeat: 'no-repeat'
         });
       }
+
+      // Don't listen anymore
+      window.removeEventListener('scroll', this.eventHandler);
```

Comment:
I don't think we should be doing this.  think about floating sidebars and it hits the bottom, we don't want it to stop.  Or when we have in/out animation (https://womenintheworkplace.com/) it runs again if you scroll back up and down

Reply:
This *only* happens for images.

Reply:
oh, phew